src/providers/user/user.ts

```diff
@@ -10,8 +10,15 @@ import { Injectable } from '@angular/core';
 @Injectable()
 export class UserProvider {
 
+  regUrl: string = "https://j-reboot-backend.herokuapp.com/api/appUsers"
```

Comment:
This regURL seems a bit too specific for everyone's use case. Is there a better way to do this? Let's look over environment variables.